services/decoder/protocols/aave-v3/aave-v3.decoders.ts

```diff
@@ -181,18 +182,21 @@ GoldRushDecoder.on(
             },
             {
                 title: "Premium",
-                value: new Intl.NumberFormat().format(
-                    Number(decoded.premium) /
-                        10 ** FlashloanToken?.[0]?.contract_decimals
-                ),
+                value:
+                    new Intl.NumberFormat().format(
+                        Number(decoded.premium) /
+                            10 ** FlashloanToken?.[0]?.contract_decimals
+                    ) +
+                    " " +
+                    FlashloanToken?.[0]?.contract_ticker_symbol,
                 type: "text",
             },
```

Comment:
The Premium should be in tokens, not details

Reply:
Your are right! fixed